keydata_test.go

```diff
@@ -1427,3 +1427,88 @@ func (s *keyDataSuite) TestReadAndWriteWithLegacySnapModelAuthKey(c *C) {
 	c.Check(err, IsNil)
 	c.Check(ok, testutil.IsTrue)
 }
+
+func (s *keyDataSuite) TestLegacyKeyData(c *C) {
```

Comment:
this needs to be mentioned in the description and summary, as it is not quite related with the rename:

Keydata v3 auxilary primary rename and legady keydata test

etc

Reply:
Changed the title and summary, thanks for pointing it out.